typescript/infra/config/environments/mainnet3/tokenPrices.json

```diff
@@ -3,9 +3,11 @@
   "arbitrum": "2914.61",
   "avalanche": "35.71",
   "bsc": "373.19",
+  "blast": "2914.61",
   "celo": "0.772619",
   "ethereum": "2914.61",
   "mantapacific": "2914.61",
+  "mode": "2914.61",
```

Comment:
this one looks under $1

Reply:
see https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3653#discussion_r1579354284